front/pages/api/poke/workspaces/[wId]/agent_configurations/[aId]/resurrect.ts

```diff
@@ -0,0 +1,79 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import {
+  getAgentConfiguration,
+  resurrectAgentConfiguration,
+} from "@app/lib/api/assistant/configuration";
+import { Authenticator, getSession } from "@app/lib/auth";
+import { apiError, withLogging } from "@app/logger/withlogging";
+
+export type ResurrectAgentConfigurationResponseBody = {
+  success: true;
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorReponse<ResurrectAgentConfigurationResponseBody>
+  >
+): Promise<void> {
+  const session = await getSession(req, res);
+  const auth = await Authenticator.fromSuperUserSession(
+    session,
+    req.query.wId as string
+  );
+
+  if (!auth.isDustSuperUser()) {
+    return apiError(req, res, {
+      status_code: 404,
+      api_error: {
+        type: "user_not_found",
+        message: "Could not find the user.",
+      },
+    });
+  }
+
+  switch (req.method) {
+    case "POST":
+      const { wId } = req.query;
+      if (!wId || typeof wId !== "string") {
+        return apiError(req, res, {
+          status_code: 400,
+          api_error: {
+            type: "invalid_request_error",
+            message: "The request query is invalid, [wId] must be set.",
+          },
+        });
+      }
+
+      const agentConfiguration = await getAgentConfiguration(
+        auth,
+        req.query.aId as string
+      );
+      if (!agentConfiguration) {
+        return apiError(req, res, {
+          status_code: 404,
+          api_error: {
+            type: "agent_configuration_not_found",
+            message: "Could not find the agent configuration.",
+          },
+        });
+      }
+
+      await resurrectAgentConfiguration(auth, agentConfiguration.sId);
```

Comment:
Should we check the return of `resurrectAgentConfiguration` 😅

Reply:
fair